src/legendhpges/invcoax.py

```diff
@@ -57,9 +35,12 @@ def _tan(a):
                 0,
                 c.taper.bottom.height_in_mm,
             ]
+            surfaces += ["n+", "n+"]
```

Comment:
I think it would be better to use strings that can be interpreted as valid variable names, i.e. `nplus`